prisma/schema.prisma

```diff
@@ -593,6 +599,33 @@ model AuthenticationAttempt {
   @@map("LoginAttempts")
 }
 
+model Passkey {
+  id            String   @id @default(uuid())
+  credentialId  String   @db.VarChar(255)
+  publicKey     String   @db.VarChar(1024)
+  signCount     Int      @default(0)
+  label         String   @db.VarChar(255)
+  createdAt     DateTime @default(now())
+  lastUsedAt    DateTime @default(now())
+
+  UserProfile   UserProfile @relation(fields: [userId], references: [supId], onDelete: Cascade)
+  userId        String      @db.Uuid
+
+  @@unique([userId, credentialId], name: "userPasskey")
+  @@map("Passkeys")
+}
+
+model PasskeyChallenge {
+  id        String   @id @default(uuid())
+  userId    String   @db.VarChar(255)
+  value     String   @db.VarChar(255)
+  version   String   @db.VarChar(255)
```

Comment:
I've seen you also set an `"email-verification"` value in here. I think you should use another enum property, similar to `ChallengePurpose` in the `Challenge` model for that, it looks like this field is currently trying to do two things at the same time. For example, this is a small code smell `version: { not: "email-verification" }`, as you don't have a specific value for `version` to query.